openhtf/core/__init__.py

```diff
@@ -133,28 +140,14 @@ def _thread_proc(self):
         # Tear down plugs first, then output test record.
         exit_stack.callback(self.test_state.plug_manager.tear_down_plugs)
 
-        # Perform initialization of some top-level stuff we need.
-        executor = self._make_phase_executor(exit_stack)
-
       # Everything is set, set status and begin test execution.  Note we don't
       # protect this with a try: block because the PhaseExecutor handles any
       # exceptions from test code.  Any exceptions here are caused by the
       # framework, and we probably want them to interrupt framework state
       # changes (like the transition to FINISHING).
+      self.test_state.mark_test_started()
```

Comment:
this has moved down fairly far, why did it move to be after plug initialization? that's a pretty significant change.  in particular, it means the test start_time_millis no longer includes plug init time, which may be nontrivial.  plug instantiation is "part of the test", so it shouldn't be excluded from the start time.